config/prometheus/monitor.yaml

```diff
@@ -5,16 +5,22 @@ kind: ServiceMonitor
 metadata:
   labels:
     control-plane: controller-manager
-  name: controller-manager-metrics-monitor
+  name: lvm-operator-controller-manager-metrics-monitor
   namespace: system
 spec:
   endpoints:
     - path: /metrics
-      port: https
-      scheme: https
+      port: topolvm-metrics
+      scheme: http
+      bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+      tlsConfig:
+        insecureSkipVerify: true
+    - path: /metrics
+      port: lvm-metrics
+      scheme: http
       bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
       tlsConfig:
         insecureSkipVerify: true
   selector:
     matchLabels:
-      control-plane: controller-manager
```

Comment:
do not delete this label, it still exists at deployment resource level above

Reply:
Ack... Adding it.
I understand that there is a label (in this `ServiceMonitor` and in other resources as well), `controller-manager`. But this is the selector, so If we are monitoring only the resources with label `app.kubernetes.io/compose: metrics` why should we add the above (deleted) label?
Or is it something like, as this is the only ServiceMonitor we should monitor the controller's label?

Reply:
@leelavg , as you have found `control-plane` key don't match the dns naming convention and needed to be removed